public/app/features/profile/UserSessions.tsx

```diff
@@ -60,6 +60,7 @@ class UserSessions extends PureComponent<Props> {
                       <Button
                         size="sm"
                         variant="destructive"
+                        tooltip={'Revoke user session'}
```

Comment:
```suggestion
                        tooltip={t('user-session.revoke', 'Revoke user session')}
```
You could use the same approach as for the aria-label. This enables translations for the string.